roles/barbican/templates/etc/uwsgi/barbican-vassals-api.ini

```diff
@@ -0,0 +1,10 @@
+[uwsgi]
+http-socket = 127.0.0.1:{{ barbican.uwsgi.http_port.public }}
```

Comment:
If we have the communication be between the  HAProxy and the uwsgi process, then we should have the http-socket bind to `0.0.0.0:9312`.
So it would look like:
`User Request --> FQDN:9311 (HAProxy) --> 0.0.0.0:9312 (uwsgi backend api port)`. So we only need 2 ports since we are removing Apache from the pipeline.